docs/fast_data/tutorials/low-code.mdx

```diff
@@ -26,348 +26,347 @@ To do that, in the console, you will:
+Once the service has been attached, you can click on the right arrow of the micro-service and edit its configuration.
 
-The Kafka topic names can be changed in the configuration, but please mind that their name must be unique, and thus cannot be reused in other projections.
+![Edit PS Configuration](img/34-edit-PS.png)
 
-:::info
-The application does not create Kafka topics for you. If you decide to use a different Kafka topic name, please be sure the topic already exists and it is fully operational.
-The Real-Time Updater, the service that receives the update from the source System, will crash at startup if there is a topic not defined in Kafka.
-:::
+In this section, you can setup the following configurations:
 
-### Order Projection
+* [Kafka Consumer](/fast_data/configuration/projection_storer.md#consumer), which will be used to read [ingestion-messages](/fast_data/inputs_and_outputs.md#ingestion-message),
+* [Kafka Producer](/fast_data/configuration/projection_storer.md#producer), which will be used to generate [pr-update messages](/fast_data/inputs_and_outputs.md#projection-update-message),
+* [Storage Configuration](/fast_data/configuration/projection_storer.md#storage), which will contain the MongoDB connection where the database is located.
 
-The second projection we will create is the _order_ one. This projection will store the data of the user's orders.
-Again, inside a System of Record page, we need to click on the button "Create new Projection", input _order_ as the projection name and click on "Create".
+</p>
+</details>
 
-![Create Order Projection](img/15-create-other-projection.png)
+<details>
+<summary>Real-Time Updater Setup</summary>
+<p>
 
-Once the projection is created, we can see the page of the _order_ Projection.
-
-![Order Projection Overview](img/16-other-projection-overview.png)
-
-#### Fields
-
-Similar to the _registry_ Projection, we ask you to add the following fields:
+:::caution
+From version `11.7.0` of the Console, the _Real-Time Updater_ service must be created manually with the guide explained in this paragraph.
 
-| Field           | Type   | Description                                                    |
-|-----------------|--------|----------------------------------------------------------------|
-| **order_id**    | String | The order ID. Check this field as "Required" and "Primary Key".|
-| **name**        | String | The name of the product.                                       |
-| **price**       | Number | The price of the product.                                      |
-| **date**        | String | The date when the order was placed.                            |
-| **registry_id** | String | The registry ID of the customer who placed the order.          |
+If you're using an older version, you can skip this paragraph since the service is automatically created when saving the configuration after creating the System of Record.
+:::
```

Comment:
Since this callout is the same of the one at the beginning of the paragraph I would remove it.